code/modules/client/preference_setup/loadout/loadout_fluffitems_vr.dm

```diff
@@ -1020,6 +1020,12 @@
 	ckeywhitelist = list("vitoras")
 	character_name = list("Roanna Ti'Rox")
 
+/datum/gear/fluff/kaboombelt
+	path = /obj/item/weapon/storage/belt/utility/full/fluffsynth
```

Comment:
Wouldn't this be behind engineering roles only, like most other loadout items that provide tools related to specific department roundtart?

Reply:
I suppose I could add that, but with the exception of the black screwdriver and garentee of black coiling everything in the belt is obtainable from public tool storage.